crates/dbsp/src/operator/differentiate.rs

```diff
@@ -35,6 +35,11 @@ where
                     &self.try_sharded_version().delay(),
                 );
                 differentiated.mark_sharded_if(self);
+
+                self.circuit().cache_insert(
+                    IntegralId::new(differentiated.origin_node_id().clone()),
```

Comment:
why are these swapped?

Reply:
Swapped wrt what?  This is supposed to say that the input of the differentiator is equal to the integral of its output.  Or, in other words, differentiation is the inverse of integration.